### PR TITLE
Add a BVT test for compliance history with Gatekeeper integration

### PR DESCRIPTION
**Pull request by mprahl**
Relates:
https://issues.redhat.com/browse/ACM-10033

### CODE REVIEW

File: test/integration/compliance_history_test.go [hunk 1]
```diff
@@ -194,86 +221,258 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
+		_, err := common.OcHub(
+			"apply",
+			"-f",
+			"../resources/compliance_history/policy-install-gk.yaml",
+		)
+		Expect(err).ToNot(HaveOccurred())
 
-			parentPolicy, err = clientHubDynamic.Resource(common.GvrPolicy).Namespace(policyNS).Get(
-				ctx, policyName, metav1.GetOptions{},
+		DeferCleanup(func(ctx context.Context) {
```
Comment (yiraeChristineKim) [inline test/integration/compliance_history_test.go:281]: DeferCleanup doesn't work when `It` failed right? so I wrapped with Describe and used `afterall`

Comment (mprahl) [reply test/integration/compliance_history_test.go:281]: I haven't noticed that issue. It looks like the docs say that it's the same as an AfterEach under the hood:
https://onsi.github.io/ginkgo/#cleaning-up-our-cleanup-code-defercleanup

**Review by yiraeChristineKim — APPROVED** [review]

**Review by yiraeChristineKim — APPROVED** [review]

**Comment by mprahl** [comment]
I'll think I'll need to wait for the next ACM snapshot for this to pass.

**Pull request merged**